iset.mm

```diff
@@ -109407,6 +110183,76 @@ seq m ( + , ( n e. ZZ |-> if ( n e. A , [_ n / k ]_ B , 0 ) ) , CC )
         TVNQWCVOPUPAVOURUSUTVAWDGUGQZVLUHQWFVMUEAWGVOKUIRSFHUJDVLGUHUKUMVBAWGVS
         VIVOVCKLGHVEVFVG $.
     $}
+
+    ${
+      isummolem3.5 $e |- ( ph -> ( M e. NN /\ N e. NN ) ) $.
+      isummolem3.6 $e |- ( ph -> f : ( 1 ... M ) -1-1-onto-> A ) $.
+      isummolem3.7 $e |- ( ph -> K : ( 1 ... N ) -1-1-onto-> A ) $.
+      $( Lemma for summation.  (Contributed by Jim Kingdon, 15-Aug-2022.) $)
```

Comment:
Lemma for ~isummo ? Same for next theorem.
(We have in our conventions that xxxlem should have a comment beginning with "Lemma for ~xxx.")

Reply:
Sure, but there isn't (yet) a theorem called isummo so the validation would fail if we worded it that way.